res/controllers/common-controller-scripts.js

```diff
@@ -32,7 +32,7 @@ String.prototype.toInt = function() {
 
 // Causes script print() calls to appear in the log file as well
 var print = function(string) {
-    engine.log(string);
+    console.log(string);
```

Comment:
This should ideally have a deprecation warning, like this:

```suggestion
    console.warn("print() is deprecated, use console.log instead!");
    console.log(string);
```

Unfortunately, there are a lot of mappings that use this. I don't think it's worth to touch that many files:

```
Behringer-CMDStudio4a-scripts.js
DJTechTools-MIDIFighter-scripts.js
Numark-Mixtrack-Pro-scripts.js
American-Audio-RADIUS-2000-scripts.js
Hercules-DJ-Control-Instinct-scripts.js
Traktor-Kontrol-S4-MK2-hid-scripts.js
Hercules-DJ-Control-MP3-hid-scripts.js
Midi_for_light-scripts.js
Traktor-Kontrol-X1.js
Soundless_joyMIDI_scripts.js
.eslintrc.json
Numark-Omni-Control-scripts.js
Reloop-Digital-Jockey2-Controller-scripts.js
Stanton-SCS1m-scripts.js
EKS-Otus.js
Stanton-SCS3d-scripts.js
Hercules-DJ-Console-4-Mx-scripts.js
Hercules DJ Control MP3 e2-scripts.js
Denon-MC3000-scripts.js
HID-Trackpad.js
Numark-Mixtrack-3-scripts.js
Behringer-CMD-Micro-scripts.js
Korg-nanoKONTROL-2-scripts.js
Numark-Mixtrack-Platinum-scripts.js
Stanton-SCS3m-scripts.js
Hercules-DJ-Console-Mk1-hid-scripts.js
common-controller-scripts.js
Gemini-CDMP-7000-scripts.js
Korg-nanoPAD2-scripts.js
Hercules-DJ-Console-RMX-hid-scripts.js
midi-components-0.0.js
Reloop-Jockey-3-ME-scripts.js
Stanton-SCS1d-scripts.js
DJ-Tech-Mixer-One-scripts.js
DJ-Tech-Kontrol-One-scripts.js
Electrix-Tweaker-scripts.js
DJ-Tech-i-Mix-Reload-scripts.js
Reloop-Beatmix-2-4-scripts.js
Hercules-DJ-Control-MP3-scripts.js
Hercules_DJControl_Jogvision-scripts.js
Reloop Terminal Mix 2-4.js
Numark-V7-scripts.js
Allen-and-Heath-Xone-K2-scripts.js
Vestax-VCI-400-scripts.js
Reloop-Beatpad-scripts.js
TrakProDJ-iPad-scripts.js
American-Audio-VMS4-scripts.js
Vestax-Spin-scripts.js
Numark-Total-Control-scripts.js
Hercules-mp3e2-compat.js
Behringer-Extension-scripts.js
Hercules-DJ-Console-RMX-scripts.js
Vestax-VCI-300-scripts.js
Stanton-SCS3d-alternate-scripts.js
lodash.mixxx.js
Akai-LPD8-RK-scripts.js
Numark-Mixtrack-2-scripts.js
Hercules-DJ-Control-Steel-scripts.js
American-Audio-VMS2-scripts.js
Ion-Discover-DJ-scripts.js
Novation-Launchpad-Mini-scripts.js
HID-Keyboard.js
Vestax-VCI-100-hile.js
KANE_QuNeo_scripts.js
Gemini-FirstMix-scripts.js
Behringer-BCD3000-Advanced-scripts.js
Behringer-BCD3000-scripts.js
Behringer-CMD-MM1-scripts.js
Denon-MC6000MK2-scripts.js
common-hid-packet-parser.js
Vestax-Typhoon-scripts.js
Behringer-BCD2000-scripts.js
Mixman DM2 (OS X).js
Hercules-DJ-Console-Mk2-scripts.js
Hercules-DJ-Control-AIR-scripts.js
```

Reply:
You could add an jsdoc directive. Code completion would highlight the deprecated fact when they're using it.
```javascript
/**
 * @deprecated use `console.log/warn/debug` instead
 */
 var print = function(string) {
```

Reply:
Kinda weird yes, but I'd like to use JSDoc for all of our JS libraries. This would be first step and justifiable under the "boyscout rule"

Reply:
I updated the code. I doubt this works due to our non-standard "import system", but it's better than nothing.

Reply:
Even if it doesn't work in code editor, it still serves as documentation.